### PR TITLE
Update link to spm ios

### PR DESCRIPTION
**Pull request by KennyHuRadar**

### CODE REVIEW

File: docs/sdk/ios.mdx [hunk 1]
```diff
@@ -49,7 +49,7 @@ Then, run `pod install`. You may also need to run `pod repo update`.
 
 ### Swift Package Manager
 
-In Xcode, go to *File* > *Swift Packages* > *Add Package Dependency*. Enter `https://github.com/radarlabs/radar-sdk-ios.git` for the *Package Repository URL*.
+In Xcode, go to *File* > *Swift Packages* > *Add Package Dependency*. Enter `https://github.com/radarlabs/radar-sdk-ios-spm.git` for the *Package Repository URL*.
```
Comment (lmeier) [inline docs/sdk/ios.mdx:52]: let's add a note calling attention to the fact that this is a distinct repo. Probably too easy to gloss past this with just this alone

Comment (lmeier) [reply docs/sdk/ios.mdx:52]: we should add a note in the readme of the ios repo as well

Comment (KennyHuRadar) [reply docs/sdk/ios.mdx:52]: the iOS repo has been updated

**Comment by KennyHuRadar** [comment]
removing as duplicate

**Pull request closed without merging**